styles.css

```
@@ -0,0 +1,680 @@
+.rolling-stone-wrapper {
+    flex-basis: 700px;
+    flex-grow: 0;
+    flex-shrink: 1;
+    padding: 50px 40px;
+}
+
+.rolling-stone-image {
+    box-shadow: 0 20px 20px rgba(0, 0, 0, 0.15)
+}
+
+
+.variety-article {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: center;
+
+}
+
+.variety-title {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 400px;
+    padding: 90px 50px 90px 90px;
+    font-weight: 600;
+    font-size: 16px;
+}
+
+.variety-wrapper {
+    flex-basis: 700px;
+    flex-grow: 0;
+    flex-shrink: 1;
+    padding: 50px 40px;
+}
+
+.variety-image {
+    box-shadow: 0 20px 20px rgba(0, 0, 0, 0.15)
+}
+
+
+.worth-article {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.worth-title {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 400px;
+    padding: 90px 90px 90px 50px;
+    font-weight: 600;
+    font-size: 16px;
+}
+
+.worth-wrapper {
+    flex-basis: 700px;
+    flex-grow: 0;
+    flex-shrink: 1;
+    padding: 50px 40px;
+}
+
+.worth-image {
+    box-shadow: 0 20px 20px rgba(0, 0, 0, 0.15)
+}
+
+
+/*--------------------- 4.4 Services -----------------*/
+.tiles-wrapper {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    flex-wrap: wrap;
+    row-gap: 25px;
+    padding: 50px 50px;
+}
+
+
+.web-design-tile {
+    border: 2px solid #5722CD;
+    display: flex;
+    flex-direction: column;
+    flex-basis: 332px;
+    padding: 32px;
+    align-items: flex-start;
+    font-size: 14px;
+    box-shadow: 0 20px 20px rgba(0, 0, 0, 0.15)
+}
+
+.web-design-tile h2{
+    padding: 10px 0 10px 0;
+    color: #5722CD;
+    font-size: 22px;
+}
+
+.web-design-tile p{
+    color: #7A7A7A;
+}
+
+.icon-web-design{
+    width: 60px;
+}
+
+
+
+.web-dev-tile {
+    border: 2px solid #5722CD;
+    display: flex;
+    flex-direction: column;
+    flex-basis: 332px;
+    padding: 32px;
+    align-items: flex-start;
+    font-size: 14px;
+    box-shadow: 0 20px 20px rgba(0, 0, 0, 0.15)
+}
+
+.web-dev-tile h2{
+    padding: 10px 0 10px 0;
+    color: #5722CD;
+    font-size: 22px;
+}
+
+.web-dev-tile p{
+    color: #7A7A7A;
+}
+
+.icon-web-dev{
+    width: 60px;
+}
+
+
+
+
+.user-ex-tile {
+    border: 2px solid #5722CD;
+    display: flex;
+    flex-direction: column;
+    flex-basis: 332px;
+    padding: 31px;
+    align-items: flex-start;
+    font-size: 14px;
+    box-shadow: 0 20px 20px rgba(0, 0, 0, 0.15)
+}
+
+.user-ex-tile h2{
+    padding: 10px 0 10px 0;
+    color: #5722CD;
+    font-size: 22px;
+}
+
+.user-ex-tile p{
+    color: #7A7A7A;
+}
+
+.icon-ux {
+    width: 60px;
+}
+
+
+
+
+
+.logo-design-tile {
+    border: 2px solid #5722CD;
+    display: flex;
+    flex-direction: column;
+    flex-basis: 332px;
+    padding: 29px;
+    align-items: flex-start;
+    font-size: 14px;
+    box-shadow: 0 20px 20px rgba(0, 0, 0, 0.15)
+}
+
+.logo-design-tile h2{
+    padding: 10px 0 10px 0;
```

Dit kun je verkorten:
```suggestion
    padding: 10px 0;
```